contrast/contrast-service.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM buildpack-deps:stretch
```

Comment:
I was curious if stretch was the common base image for utils, and it looks like there's no real standard across the dockerfiles in the repo. 👍   on going with one with a good tradeoff for image size versus extra libs needed.

Reply:
Yeah, we could alter this in the future, but stretch was the base for the python docker files I was working on at the time, so I used stretch.